### PR TITLE
Added MovingPictures data set

### PR DESCRIPTION
**Pull request by antagomir**

### CODE REVIEW

**Comment by antagomir** [comment]
I made the PR to microbiome/mia - closing this now.

**Comment by FelixErnst** [comment]
Are you sure about this? I think this will end up in a mess, if this is the git strategy you want to pursue.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 .Rhistory
 .RData
 .Ruserdata
+inst/
+*~
```
Comment (FelixErnst) [inline .gitignore:6]: Which files get removed by this?

File: R/makeMicrobiomeExperimentFromphyloseq.R [hunk 2]
```diff
@@ -17,6 +17,12 @@
 #'     makeMicrobiomeExperimentFromphyloseq(enterotype)
 #'     data(esophagus, package="phyloseq")
 #'     makeMicrobiomeExperimentFromphyloseq(esophagus)
+#'
```
Comment (FelixErnst) [inline R/makeMicrobiomeExperimentFromphyloseq.R:20]: Examples should be run. What is the intent here? The fix should be applied inside `makeMicrobiomeExperimentFromphyloseq`. Please suggest a general fix for this function

**Review by FelixErnst — COMMENTED** [review]
Hi Leo @antagomir,  

I have not seen the MovingPictures dataset previously. Can you briefly summarize the added value?

Thanks,
Felix

**Pull request closed without merging**